functions/src/cron/test.http

```diff
@@ -0,0 +1 @@
+GET http://localhost:5000/instantvisio-dev/us-central1/api/api/v1-tests/reminders/runCron
```

Comment:
are you following a pattern for these `.http` files ?

Reply:
yes, it directly integrated in mY IDE so I can run http request in the same window

Reply:
this one will be removed before merge though